pom.xml

```diff
@@ -89,7 +89,7 @@
 			<dependency>
 				<groupId>org.openmrs.module</groupId>
 				<artifactId>metadatamapping-api</artifactId>
-				<version>1.0.1</version>
+				<version>1.1.0-alpha2-SNAPSHOT</version>
```

Comment:
Any reason you changed to SNAPSHOT and not 1.1.0-alpha1?

Also the new required version of metadatamapping module needs to be set in config.xml